assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -26,7 +26,13 @@ const calcPriceDifference = ( subtotal, total ) => {
 
 const ProductVariationDetails = ( { variation } ) => {
 	const variationsText = variation
-		.map( ( v ) => `${ v.attribute }: ${ v.value }` )
+		.map( ( v ) => {
+			if ( v.attribute ) {
+				return `${ v.attribute }: ${ decodeEntities( v.value ) }`;
+			}
+			// Support for product attributes with no name/key
```

Comment:
When does this happen?

Reply:
I initially saw this with @LevinMedia store test data - the product sizes/dimensions were an unnamed attribute.

Also in the Woo sample product data I saw it with `Hoodie`, the logo attribute name is missing. 

Here's what this looks like without the fix, note the extraneous `:`. Seeing this I assumed that it's supported to have attribute values that are self-contained, is this supported in woo?

<img width="693" alt="Screen Shot 2020-02-18 at 11 41 31 AM" src="https://user-images.githubusercontent.com/4167300/74690510-3c3b6280-5244-11ea-90f0-18ef8abc2032.png">

Hmm.. looking at the [docs](https://docs.woocommerce.com/document/variable-product/) it seems attributes should have a name. I'll investigate further, maybe this is getting lost in the API response.

Reply:
OK mystery solved! This happens with `Variable products`. I set up a test product - banana, available as curved or straight. The attribute name is missing from the API in this case, which is consistent with how this product is rendered in existing (shortcode) cart. So I think this code is good to go as is :)

<img width="502" alt="Screen Shot 2020-02-18 at 12 12 51 PM" src="https://user-images.githubusercontent.com/4167300/74691609-6000a780-5248-11ea-9adf-552da9490063.png">

<img width="500" alt="Screen Shot 2020-02-18 at 12 12 59 PM" src="https://user-images.githubusercontent.com/4167300/74691617-62630180-5248-11ea-84b1-84ec98b88293.png">

<img width="1096" alt="Screen Shot 2020-02-18 at 12 12 37 PM" src="https://user-images.githubusercontent.com/4167300/74691624-6727b580-5248-11ea-8f47-fe72fecc690a.png">

<img width="838" alt="Screen Shot 2020-02-18 at 12 12 15 PM" src="https://user-images.githubusercontent.com/4167300/74691636-6bec6980-5248-11ea-97d7-12623084cd2d.png">

<img width="686" alt="Screen Shot 2020-02-18 at 12 12 24 PM" src="https://user-images.githubusercontent.com/4167300/74691639-6ee75a00-5248-11ea-84d2-f5c811a1c0b2.png">